app/imports/components/mainLayout.jsx

```diff
@@ -0,0 +1,14 @@
+import React from 'react'
+
+import Header from '/imports/components/header.jsx';
+import Footer from '/imports/components/footer.jsx';
+
+export const MainLayout = ({header, content, footer}) => (
```

Comment:
Main layout is the only component addressed in the router. Header and Footer are not used anymore i think… gotta check